index.d.ts

```diff
@@ -249,7 +249,7 @@ declare module 'warframe-items' {
     patchlogs?: PatchLog[];
     rarity?: Rarity;
     tradable: true;
-    type: 'Arcane';
+    type: 'Arcane' | 'Primary Arcane' | 'Secondary Arcane' | 'Melee Arcane' | 'Warframe Arcane' | 'Amp Arcane' | 'Operator Arcane'| 'Zaw Arcane' | 'Kitgun Arcane' | 'Bow Arcane'  | 'Shotgun Arcane' | 'Sniper Arcane';
```

Comment:
make a compound type

```suggestion
    type: 'Arcane' | `${ArcaneType} Arcane`;
```

and then elsewhere
```ts
type ArcaneType = 'Primary' | 'Secondary' | 'Melee' | 'Warframe' | 'Amp' | 'Operator' | 'Zaw' | 'Bow' | 'Shotgun' | 'Sniper';
```
that way it's faster to add just a new "ArcaneType"

the second one probably needs linting, too